tools/cli/cliRouter.js

```diff
@@ -42,7 +41,7 @@ export async function cli() {
 	argv
 		.showHelpOnFail( true )
 		.demandCommand()
-		.strictCommands()
+		.recommendCommands()
```

Comment:
What's your thought on dropping `strictCommands()`. They do work together, e.g. `jetpack dockers` will suggest `docker` while `jetpack regieergear` will exit with no output without `strictCommands`. With it, it'll tell me it is an unsupported command with the output of the help text.